debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -382,6 +391,23 @@ private static int validateMaxQueueSize(Configuration config, Field field, Field
         return count;
     }
 
+    private static int validateSkipOperation(Configuration config, Field field, ValidationOutput problems) {
+        String operations = config.getString(field);
+        for (String operation : operations.split(",")) {
+            switch (operation) {
+                case "create":
+                case "update":
+                case "delete":
+                    continue;
+                default:
+                    problems.accept(field, operation, "Invalid oplog operation");
```

Comment:
Shouldn't speak about "oplog", as that's specific to MongoDB.

Reply:
Yes. Thanks i'm going to fix these errors